src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -449,7 +450,6 @@ const formConfig = {
               items: {
                 privateRecords: Object.assign({},
                   fileUploadUI('Upload your private medical records', {
-                    allowRename: true,
```

Comment:
Looks like this was changed to use `attachmentName`, but that's not used here. Was this intentional?

Reply:
This is intentional, attachmentName is now being used instead to determine whether that field should be displayed.